mungegithub/mungers/submit-queue.go

```diff
@@ -1077,10 +1080,12 @@ func (sq *SubmitQueue) validForMergeExt(obj *github.MungeObject, checkStatus boo
 		}
 	}
 
-	// PR cannot have the label which prevents merging.
-	if obj.HasLabel(doNotMergeLabel) {
-		sq.SetMergeStatus(obj, noMerge)
-		return false
+	// PR cannot have any labels which prevent merging.
+	for _, label := range obj.LabelSet().List() {
+		if strings.HasPrefix(label, doNotMergePrefix) {
```

Comment:
Given our intent to migrate to tide I would prefer we list each one explicitly rather than matching the prefix. If this is annoying it is an early indication that the solution is probably not the right one.

It also raises the cost of adding new `do-not-merge/NEW_REASON` logic, which helps reduce the complexity of the system.

Reply:
SGTM as well